GDCpp/GDCpp/Extensions/Builtin/RuntimeSceneTools.cpp

```diff
@@ -3,13 +3,13 @@
  * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
  * reserved. This project is released under the MIT License.
  */
-#include "GDCpp/Extensions/Builtin/RuntimeSceneTools.h"
 
 #include <iostream>
 #include <vector>
 
 #include "GDCore/Tools/Log.h"
 #include "GDCpp/Extensions/Builtin/CommonInstructionsTools.h"
+#include "GDCpp/Extensions/Builtin/RuntimeSceneTools.h"

```

Comment:
This will probably be changed back by clang-format :) Check in RuntimeSceneTools, if it does not compile, it means that a header is missing in RuntimeSceneTools.h
The rule of thumb is that headers should compile whatever their order, so they need to have all the needed `#include`